src/main/resources/dbreplica/migration/V1.0.2__init_report_db.sql

```diff
@@ -136,6 +136,10 @@ create table cft_task_db.reportable_task
     handling_time                   INTERVAL,
     processing_time                 INTERVAL,
     due_date_to_completed_diff_time INTERVAL,
+    state_label              TEXT,
+    role_category_label      TEXT,
+    jurisdiction_label       TEXT,
+    case_type_label          TEXT,
```

Comment:
I think in the specification the state/jurisdiction/case_type should never not null.

Reply:
This needs to be in it's own migration version as an alter statement.  I won't work in AAT like this.